buildpacks/rust/bin/detect

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+if [[ ! -f Cargo.toml ]] ; then
```

Comment:
Soon I think we should change all of the buildpacks to look first for a func.yaml file.

Reply:
if there wasn't a func.yaml file, how would func know which builder to even run?

Reply:
Exactly - and if there's not a func.yaml, then it's probably not a function project

Reply:
I'm thinking it's a redundant, superfluous check because of course there's a func.yaml there because that's the only possible way `func` would've invoked this builder in the first place...

But you must be thinking that the buildpack might be invoked by something other than `func` so we need to check for `func.yaml` to prevent... what exactly? Sorry for being dense.